### PR TITLE
docs: rework README, add page about ignoring diagnostics

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by lucacasonato — APPROVED** [review]
LGTM

File: www/public/ignoring-rules.md [hunk 1]
```diff
@@ -1,9 +1,60 @@
...
+
+```ts
+// deno-lint-ignore-file
+
+function foo(): any {
+  // ...
+}
+```
+
+Ignore directive must be placed before first stament or declaration:
```
Comment (RDambrosio016) [inline www/public/ignoring-rules.md:15]:
```suggestion
The ignore directive must be placed before the first statement or declaration:
```

File: www/public/ignoring-rules.md [hunk 2]
```diff
@@ -1,9 +1,60 @@
-## foo bar
+## Ignore directives
 
-Hello World
+### Files
+
+To ignore whole file use `// deno-lint-ignore-file` directive at the top of the file.
```
Comment (RDambrosio016) [inline www/public/ignoring-rules.md:5]:
```suggestion
To ignore a whole file use `// deno-lint-ignore-file` at the top of the file:
```

File: www/public/ignoring-rules.md [hunk 3]
```diff
@@ -1,9 +1,60 @@
...
+// deno-lint-ignore-file no-explicit-any no-empty
+
+function foo(): any {
+  // ...
 }
 ```
+
+### Diagnostics
+
+To ignore specific diagnostic use `// deno-lint-ignore <codes...>` directive on the preceding
```
Comment (RDambrosio016) [inline www/public/ignoring-rules.md:45]:
```suggestion
To ignore specific diagnostics use `// deno-lint-ignore <codes...>` on the preceding
```

File: www/public/ignoring-rules.md [hunk 4]
```diff
@@ -1,9 +1,60 @@
...
+  // ...
+}
+
+// deno-lint-ignore no-explicit-any explicit-function-return-type
+function bar(a: any) {
+  // ...
+}
+```
+
+Specyfing rule name that should be ignored is required.
```
Comment (RDambrosio016) [inline www/public/ignoring-rules.md:60]:
```suggestion
You must specify the names of the rules to be ignored.
```

File: www/public/ignoring-rules.md [hunk 5]
```diff
@@ -1,9 +1,60 @@
...
+// deno-lint-ignore-file
+
+import { bar } from "./bar.js";
+
+function foo(): any {
+  // ...
+}
+```
+
+You can also ignore certain diagnostics in the whole file
```
Comment (RDambrosio016) [inline www/public/ignoring-rules.md:33]:
```suggestion
You can also ignore certain diagnostics in the whole file:
```

File: www/public/ignoring-rules.md (hunk 1 above)
Comment (bartlomieju) [reply www/public/ignoring-rules.md]: Thanks @RDambrosio016, I applied your suggestions

**Pull request merged**